controllers/news.controllers.js

```diff
@@ -15,9 +16,13 @@ exports.getTopics = (req, res) => {
 
 exports.getArticleById = (req, res, next) => {
   const { article_id } = req.params;
-  selectArticleById(article_id)
-    .then((article) => {
-      res.status(200).send({ article: article });
+
+  Promise.all([selectArticleById(article_id), commentsByArticle(article_id)])
+    .then((results) => {
+      const article = results[0];
+      const { count } = results[1];
+      article.comment_count = Number(count);
```

Comment:
You shouldn't make another query just to check the comment_count. Have a look at aggregate functions (hint: count) and using joins. You should be able to make only _one_ query in the models for this which will give you all the article information PLUS the comment_count.